Makefile

```diff
@@ -81,6 +81,10 @@ generate-cli-reference:
 prepare-release: cross
 	./scripts/prepare-release.sh
 
+.PHONY: setup-openshift-ci
+setup-openshift-ci:
```

Comment:
If you look at all the other commands with `Makefile`, all of the commands can be used locally by the user / developer.

This command is related directly to CI / something that's outside of the control of the developer / user.

I wouldn't add this to Makefile and instead you should tell Prow / whatever-CI-we-use to simple run `./openshift-ci/scripts/configure-cluster.sh`

Reply:
> If you look at all the other commands with `Makefile`, all of the commands can be used locally by the user / developer.
> 
> This command is related directly to CI / something that's outside of the control of the developer / user.
> 
> I wouldn't add this to Makefile and instead you should tell Prow / whatever-CI-we-use to simple run `./openshift-ci/scripts/configure-cluster.sh`

Or maybe rename it.  This could technically be used for any OpenShift cluster brought up with installer

Reply:
Renamed to something more reasonable. Note this is not just openshift ci but can be used for any cluster brought up using openshift cluster